intrahost.py

```diff
@@ -1109,13 +1109,13 @@ def sampleIDMatch(inputString):
         Given a sample name in the form of [sample] or [sample]-#,
         return only [sample]
     """
-    idRegex = re.compile("(.*?)(?:-\d+|$)+")
+    idRegex = re.compile(r"(.*?)(?:-\d+|$)+")
     m = idRegex.match(inputString)
 
     if m:
         return m.group(1)
     else:
-        raise LookupError("The ID was not of the form (.*?)(?:-\d+|$)+, ex. 5985-0")
+        raise LookupError(r"The ID was not of the form (.*?)(?:-\d+|$)+, ex. 5985-0")
```

Comment:
Prefixing with an 'r' to indicate a [raw string literal](https://docs.python.org/2.0/ref/strings.html) seemed better than properly escaping the backslash (this is pylint-inspired).